components/dashboard/src/index.css

```diff
@@ -61,8 +61,13 @@
         @apply cursor-default opacity-50 pointer-events-none;
     }
 
-    a.gp-link {
-        @apply text-blue-500 hover:text-blue-600  dark:text-blue-400 dark:hover:text-blue-500;
+    button.gp-link {
+        @apply bg-transparent hover: bg-transparent p-0 rounded-none;
```

Comment:
wait a minute... where does this extra white-space between `hover:` and `bg-transparent` come from 🤔 

fixing it now (also in the next selector)

Reply:
@gtsiolis should be fixed now. Sorry for the hiccup, I don't really know how it happened 😮